v4/source/system.yaml

```diff
@@ -1004,3 +1004,77 @@
                 format: binary
         "404":
           $ref: "#/components/responses/NotFound"
+  /warn_metrics/status:
+    get:
+      tags:
+        - system
+      summary: Get the warn metrics status (enabled or disabled)
+      description: >
+        Get the status of a set of metrics (enabled or disabled) from the Systems table.
+
+        The returned JSON contains the metrics that we need to warn the admin on with regard to their status (we return the ones whose status is "true", which means that they are in a "warnable" state - e.g. a threshold has been crossed or some other condition has been fulfilled).
+
+        __Minimum server version__: 5.25
```

Comment:
The 5.25 version is going to be a quality release, this is going to be in 5.25 or is going to be in 5.26?